templates/grpc-server.hsfiles

```diff
@@ -0,0 +1,102 @@
+{-# START_FILE {{name}}.cabal #-}
+name:                {{name}}
+version:             0.1.0.0
+-- synopsis:
+-- description:
+homepage:            https://github.com/{{github-username}}{{^github-username}}githubuser{{/github-username}}/{{name}}#readme
+author:              {{author-name}}{{^author-name}}Author name here{{/author-name}}
+maintainer:          {{author-email}}{{^author-email}}example@example.com{{/author-email}}
+copyright:           {{copyright}}{{^copyright}}{{year}}{{^year}}2019{{/year}} {{author-name}}{{^author-name}}Author name here{{/author-name}}{{/copyright}}
+category:            {{category}}{{^category}}Web{{/category}}
+build-type:          Simple
+cabal-version:       >=1.10
+extra-source-files:  README.md
+
+executable {{name}}
+  hs-source-dirs:      src
+  main-is:             Main.hs
+  other-modules:       Schema
+  default-language:    Haskell2010
+  build-depends:       base >= 4.12 && < 5,
+                       text,
+                       mu-schema,
+                       mu-rpc,
+                       mu-protobuf,
+                       mu-grpc-server
+
+{-# START_FILE stack.yaml #-}
+resolver: lts-14.16
+extra-deps:
+# mu
+- mu-schema-0.1
+- mu-rpc-0.1
+- mu-protobuf-0.1
+- mu-grpc-server-0.1
+# dependencies of mu
+- http2-client-0.9.0.0
+- http2-grpc-types-0.5.0.0
+- http2-grpc-proto3-wire-0.1.0.0
+- warp-grpc-0.2.0.0
+- proto3-wire-1.0.0
+- language-protobuf-1.0
+
+{-# START_FILE Setup.hs #-}
+import Distribution.Simple
+main = defaultMain
+
+{-# START_FILE .gitignore #-}
+.stack-work/
+stack*.yaml.lock
+*~
+
+{-# START_FILE README.md #-}
+# {{name}}
+
+{-# START_FILE {{name}}.proto #-}
+
+syntax = "proto3";
+import "google/protobuf/empty.proto";
+
+package {{name}};
+
+service Service {
+
+}
+
+{-# START_FILE src/Schema.hs #-}
+{-# language DataKinds             #-}
+{-# language DeriveAnyClass        #-}
+{-# language DeriveGeneric         #-}
+{-# language DuplicateRecordFields #-}
+{-# language FlexibleContexts      #-}
+{-# language FlexibleInstances     #-}
+{-# language MultiParamTypeClasses #-}
+{-# language PolyKinds             #-}
+{-# language TemplateHaskell       #-}
+{-# language TypeOperators         #-}
+module Schema where
+
+import Data.Text as T
+import GHC.Generics
+import Mu.Quasi.GRpc
```

Comment:
After this line I needed to add: `import Mu.Schema` otherwise `HasSchema` is not found!

Reply:
D'oh!